app/main.py

```diff
@@ -1,2 +1,110 @@
+from typing import Any
+
+
+class Node:
+    def __init__(
+            self, key: Any,
```

Comment:
place each parameter on a new line. Key cannot be `Any`, it must be `Hashable`
Fix in all places